src/blocks/ImageTextBlock/index.tsx

```diff
@@ -28,75 +28,62 @@ import { BackgroundVideo } from './BackgroundVideo'
 
 type TitleSize = 'sm' | 'lg'
 
-const AlignableContentWrapper = styled(ContentWrapper)(
-  ({ textPosition }: { textPosition: string }) => ({
-    display: 'flex',
-    flexDirection:
-      textPosition === 'right'
-        ? 'row-reverse'
-        : textPosition === 'center'
-        ? 'column'
-        : 'row',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    [TABLET_BP_DOWN]: {
-      flexDirection: 'column',
-    },
-  }),
-)
+const AlignableContentWrapper = styled(ContentWrapper)<{
+  textPosition: string
```

Comment:
Should this use the `TextPosition`-type? 🙂

Reply:
same about this one, react complains about the camel case and suggests this name 🤷‍♀️